mne/io/cnt/cnt.py

```diff
@@ -0,0 +1,389 @@
+"""Conversion tool from Neuroscan CNT to FIF
+"""
+
+# Author: Jaakko Leppakangas <jaeilepp@student.jyu.fi>
+#
+# License: BSD (3-clause)
+from os import path
+import datetime
+import calendar
+
+import numpy as np
+
+from ...utils import warn, verbose
+from ..constants import FIFF
+from ..utils import _mult_cal_one, _find_channels, _create_chs
+from ..meas_info import _empty_info
+from ..base import _BaseRaw, _check_update_montage
+from ..utils import read_str
+
+
+def read_raw_cnt(input_fname, montage, eog=(), ecg=(), emg=(), misc=(),
```

Comment:
should we keep the signature pattern? for this to be consistent with the other eeg modules, it should go `eog, misc, ...` cc: @agramfort